src/jit/importer.cpp

```diff
@@ -6087,24 +6087,13 @@ var_types           Compiler::impImportCall (OPCODE         opcode,
                 if (opts.IsReadyToRun())
                 {
                     call->gtCall.gtEntryPoint = callInfo->codePointerLookup.constLookup;
-                    if (canTailCall)
-                    {
-                        canTailCall = false;
-                        szCanTailCallFailReason = "Ready To Run";
-                    }
                 }
 #endif
                 break;
             }
 
         case CORINFO_CALL_CODE_POINTER:
-            {     
-                if (compIsForInlining())
```

Comment:
This is checked in impLookupToTree already.

For CoreCLR, this check is equivalent to the check in impLookupToTree.
For CoreRT, this check is too broad - restricting inlining for no good reason.